content/zh/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -482,9 +482,9 @@ spec:
   - from:
     - source:
       notRequestPrincipals: ["*"]
-      to:
+    to:
```

Comment:
https://github.com/istio/istio.io/blob/15cfeb23a6f7758247ecf17bfd3916e7466e45b8/content/en/docs/tasks/security/authentication/authn-policy/index.md#L497-L513

indent still error.

`source` and `to` should have same indents.